xwiki-platform-core/xwiki-platform-annotation/xwiki-platform-annotation-ui/src/main/resources/AnnotationCode/Macros.xml

```diff
@@ -356,6 +356,8 @@
 
 #macro(displayAnnotationProperty $annotatedDocument $annotationObject $annotationProperty $mode)
   #if ($mode == 'edit' || $mode == 'create')
+    ## This startupFocus parameter is used by the CKEditor WYSIWYG editor.
+    #set ($startupFocus = true)
```

Comment:
I'm wondering if we shouldn't make his a bit more generic / extensible to prepare the ground for other configuration parameters we may need in the future:

```suggestion
    #set ($wysiwygEditorConfig = {
      'startupFocus': true
    })
```

This will make the configuration more explicit, and it will reduce the risk of naming conflicts with other entries from the script context. WDYT?